python/ambassador/ir/irhost.py

```diff
@@ -363,18 +380,20 @@ def matches_httpgroup(self, group: 'IRHTTPMappingGroup') -> bool:
             host_match = True
             self.logger.debug("-- hostname %s group regex => %s", self.hostname, host_match)
         else:
-            group_glob = group.get('host') or None
+            # It is NOT A TYPO that we use group.get("host") here -- whether the Mapping supplies
+            # "hostname" or "host", the Mapping code normalizes to "host" internally.
```

Comment:
gross :P

Reply:
Yeah. It saved an _insane_ amount of code churn at the time, but it's probably time to cause that churn for everyone's sanity.